site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -167,6 +222,33 @@ gcloud container node-pools create windows \
 
 where [CLUSTER_NAME] is the name of the cluster you created.
 
+### Create an Autopilot mode cluster for Agones
+
+Agones supports Autopilot clusters in [alpha]({{<ref "/docs/Guides/feature-stages#alpha">}}).
+
+```bash
+gcloud container clusters create-auto [CLUSTER_NAME] \
+  --region=[COMPUTE_REGION] \
+  --release-channel=[RELEASE_CHANNEL] \
+  --autoprovisioning-network-tags=game-server \
+```
+Replace the following:
+
+* **CLUSTER_NAME**: the name of your cluster.
+* **COMPUTE_REGION**: the
+  [Google Cloud region](https://cloud.google.com/compute/docs/regions-zones#available)
+  for the cluster.
+* **RELEASE_CHANNEL**: the
+  [GKE release channel](https://cloud.google.com/kubernetes-engine/docs/release-notes#current_versions)
+  for the cluster. Can be one of `rapid`, `regular`, or `stable`. The default
+  is `regular`. The release channel must use a
+  [supported Kubernetes version]({{<ref "/docs/Installation#usage-requirements">}}).
+
+In Autopilot clusters, GKE manages the nodes, so you won't create dedicated
+node pools for Agones system components or for metrics collection. The node
+affinity rules in the Agones installation manifests tell GKE to separate system
+and metrics Pods onto dedicated nodes automatically.
```

Comment:
> The node
affinity rules in the Agones installation manifests tell GKE to separate system
and metrics Pods onto dedicated nodes automatically.

This part is opposite of what I intended in the UG as well. I might just leave this part out for now - I'm really thinking we need a guide page rather than dumping all of this on the install page, and I have some stuff to say about separation at a greater depth, but it seems cluttery to put it here.

Reply:
Does that effect not happen then? I noticed the node tolerations in the manifests

Reply:
Removed

Reply:
The problem is that for pod anti-affinity, you have to have >0.25 vCPU pods, so I backed off on using them in general. The tolerations are still there, I think, but I don't think they're doing anything right now.